schemas/src/elements/inputs/Input.json

```diff
@@ -24,6 +24,16 @@
 			"type": "string",
 			"description": "Label for this input",
 			"version": "1.3"
+		},
+		"labelPosition": {
+			"type": "InputLabelPosition",
+			"description": "Label position hint for input",
```

Comment:
would this sentence be clearer - > "input label hint position" ? Also,  should we specify sample values like above/default like you did for labelWidth ?